core/database/src/main/java/com/gkp/core/database/entity/UpdatedAgendaItemEntity.kt

```diff
@@ -1,11 +1,14 @@
 package com.gkp.core.database.entity
 
+import androidx.room.Embedded
 import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 @Entity
 data class UpdatedAgendaItemEntity(
     @PrimaryKey
-    val id: String,
+    val agendaItemId: String,
     val userId: String,
+    @Embedded
+    val agendaItem: AgendaItemEntity
 )
```

Comment:
When updating events, how are you keeping track of the photos the user has deleted and the new ones that need to be uploaded? 🤔 Could it make more sense to use separate tables (Entity classes) for each agenda item?